LICENSES/GNU-AGPL-V3.md

```
@@ -0,0 +1,651 @@
+GNU Affero General Public License
+=================================
+
+_Version 3, 19 November 2007_
+_Copyright � 2007 Free Software Foundation, Inc. &lt;<http://fsf.org/>&gt;_
+
+Everyone is permitted to copy and distribute verbatim copies
+of this license document, but changing it is not allowed.
+
+## Preamble
```

markdownlint-cli2 (0.21.0)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)